spec/library_spec.rb

```diff
@@ -0,0 +1,40 @@
+require './lib/library'
+
+describe Library do
+  items = [{item:{ title: 'Alfons leker', author: 'A. Andersson' }, available: true, return_date: nil},
+           {item: { title: 'Bible', author: 'TGAOTU' }, available: true, return_date: nil} ]
+  subject { described_class.new(items: items) }
+
+  let(:user) { double('User') }
+
+  it 'has a collection of books' do
+    expect(subject.items).to be_kind_of Array
+  end
+
+  it 'finds item by #title' do
+    response = {item:{ title: 'Alfons leker', author: 'A. Andersson' }, available: true, return_date: nil}
+    expect(subject.find(title: 'Alfons leker')).to eq response
+  end
+
+  it 'finds item by #author' do
+    response = {item: { title: 'Bible', author: 'TGAOTU' }, available: true, return_date: nil}
```

Comment:
Metrics/LineLength: Line is too long. [94/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)